docs/UserGuide.md

```diff
@@ -114,20 +114,20 @@ This detailed guide will walk you through how to make the most of **EduManage**
+|   4   |    Lesson Time    |                `lt/LESSON_TIME`                | Should be in format `DAY-STARTTIME-ENDTIME`, where:<ul><li>`DAY` is in: [MON, TUE, WED, THU, FRI, SAT, SUN].</li><li>`STARTTIME` and `ENDTIME` are in the format `HH:MM`, `HH` must be between 0 and 23 inclusive, `MM` must be between 0 and 59 inclusive.</li><li>`ENDTIME` must be later than `STARTTIME`.</li></ul>                                                                                                                                                                                                                                                                                                                                     |
+|   5   |       Level       |                   `l/LEVEL`                    | Should be in the format `YEAR TRACK`, where:<ul><li>either:<ul><li>`YEAR` is one of: [S1, S2, S3, S4] and</li><li>`TRACK` is one of: [EXPRESS, NA, NT, IP].</li></ul><li>or:<ul><li>`YEAR` is S5 and</li><li>`TRACK` is NA.</ul></ul>                                                                                                                                                                                                                                                                                                                                                                                                                       |
+|   6   |       Name        | `n/NAME` or `update NAME` or `find n/KEYWORDS` | Should only contain alphanumeric characters and spaces, and should not be blank. For commands other than `add n/NAME`, `update [n/NAME]` and `find n/KEYWORDS`, the provided name must match an existing student's name in EduManage. Refer to [Section 5 - Known Issues](#5-known-issues) for guidance on handling valid names that contain special characters.                                                                                                                                                                                                                                                                                            |
+|   7   |       Note        |                   `nt/NOTE`                    | Can take any values except prefixes already reserved, and can be blank.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
Prefixes not allowed:
* `update`: `a/`, `e/`, `lt/`, `l/`, `n/`, `p/`, `s/`
* `note`: `n/`